files/scripts/profiling/deployment_time_profiling.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import pandas as pd
+from datetime import datetime
+import sys
+
+def help():
+    print("\nThe script requires exactly 2 arguments in sequence\n1.Input file\n2.Output file")
+    print("\nSample command:\n $ python time-profile-script.py input output")
+args = sys.argv
+if len(args) == 2 and (args[1] == '-h' or args[1] == '--help'):
+    help()
+    exit()
+
+if len(sys.argv) != 3:
+    help()
+    raise Exception("Please provide 2 arguments")
+
+# Reading log file and returning list of lines in the file.
+def read_file():
+    log_file_path=input("Enter path of the log_file: ")
+    log_file = open(log_file_path,"r")
+    file_list = log_file.readlines()
+    log_file.close()
+    return file_list
+
+# Parse the timestamp from the string.
+def get_formatted_time(line):
+    date=line[:23]
+    text=line[33:-1]
+    return datetime.strptime(date,'%Y-%m-%d %H:%M:%S,%f'),text
+
+
+# Print the output in a tabular format.
+def display_result(components,time_taken):
+    print("\n{:<100} | {}".format("Components","Time taken in seconds"))
+    print("{}".format("-"*125))
+    for i in range(len(time_taken)):
+        print("{:<100} | {}".format(components[i],time_taken[i]))
+        print("{}".format("-"*125))
+
+
+file_list = read_file()										# Read the input file.
+
+components = []
+time_taken = []
+
+count = 0
+prev_time = None
+
+for line in file_list:										# Looping through the lines in the file
+	if line[0].isdigit():									# Checking if the line contains a timestamp
```

Comment:
This is cryptic. What if the filename has a digit in it?
Prefer using a regex instead.